Compilation error in 0.3.x

When trying to install any of 0.3.0 to 0.3.3, I receive the following compilation error. This is not the case for 0.2.0.

OS: OS X 10.10.4
Node: 0.12.7
npm: 2.12.1

```
> node-gyp configure && node-gyp build

gyp info it worked if it ends with ok
gyp info using node-gyp@2.0.1
gyp info using node@0.12.7 | darwin | x64
gyp info spawn python
gyp info spawn args [ '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/private/tmp/node_modules/node-avro-io/node_modules/int64-native/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/daniel/.node-gyp/0.12.7/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/daniel/.node-gyp/0.12.7',
gyp info spawn args   '-Dmodule_root_dir=/private/tmp/node_modules/node-avro-io/node_modules/int64-native',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@2.0.1
gyp info using node@0.12.7 | darwin | x64
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/Int64/src/main.o
In file included from ../src/main.cc:8:
../src/Int64.h:10:22: error: expected class name
class Int64 : public ObjectWrap {
                     ^
../src/Int64.h:22:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> New(const Arguments& args);
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:23:39: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ToNumber(const Arguments& args);
                                      ^~~~~~~~~
                                      v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:24:38: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ValueOf(const Arguments& args);
                                     ^~~~~~~~~
                                     v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:25:39: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ToString(const Arguments& args);
                                      ^~~~~~~~~
                                      v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:26:54: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ToUnsignedDecimalString(const Arguments& args);
                                                     ^~~~~~~~~
                                                     v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:27:52: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ToSignedDecimalString(const Arguments& args);
                                                   ^~~~~~~~~
                                                   v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:28:37: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Equals(const Arguments& args);
                                    ^~~~~~~~~
                                    v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:29:38: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Compare(const Arguments& args);
                                     ^~~~~~~~~
                                     v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:30:37: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> High32(const Arguments& args);
                                    ^~~~~~~~~
                                    v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:31:36: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Low32(const Arguments& args);
                                   ^~~~~~~~~
                                   v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:32:40: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ShiftLeft(const Arguments& args);
                                       ^~~~~~~~~
                                       v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:33:41: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> ShiftRight(const Arguments& args);
                                        ^~~~~~~~~
                                        v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:34:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> And(const Arguments& args);
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:35:33: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Or(const Arguments& args);
                                ^~~~~~~~~
                                v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:36:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Xor(const Arguments& args);
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:37:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Add(const Arguments& args);
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
In file included from ../src/main.cc:8:
../src/Int64.h:38:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Sub(const Arguments& args);
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/daniel/.node-gyp/0.12.7/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
18 errors generated.
make: *** [Release/obj.target/Int64/src/main.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Darwin 14.4.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build"
gyp ERR! cwd /private/tmp/node_modules/node-avro-io/node_modules/int64-native
gyp ERR! node -v v0.12.7
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok
```

I'd like to chime in as a Windows user

OS: Windows 7
NPM: 2.11.3
Node: 0.12.7

```
D:\sample\sample>npm install node-avro-io
npm WARN package.json sample@0.0.1 No repository field.
npm WARN package.json sample@0.0.1 No README data
npm WARN package.json sample@0.0.1 No license field.
\
> int64-native@0.3.2 install D:\sample\node_modules\node-avro-io\node_modules\int64-native
> node-gyp configure && node-gyp build


D:\sample\node_modules\node-avro-io\node_modules\int64-native>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" configure )  else (node  configure )

D:\sample\node_modules\node-avro-io\node_modules\int64-native>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" build )  else (node  build )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  main.cc
  Int64.cc
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(10): error C2504: 'ObjectWrap' : base class undefined [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(22): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(22): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(23): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(23): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(24): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(24): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(25): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(25): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(26): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(26): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(27): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(27): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(28): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(28): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(29): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(29): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(30): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(30): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(31): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(31): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(32): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(32): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(33): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(33): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(34): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(34): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(35): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(35): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(36): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(36): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(37): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(37): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(38): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(38): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\xlocale(323): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(10): error C2504: 'ObjectWrap' : base class undefined [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(22): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(22): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(23): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(23): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(24): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(24): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(25): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(25): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(26): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(26): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(27): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(27): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(28): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(28): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(29): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(29): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(30): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(30): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(31): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(31): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(32): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(32): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(33): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(33): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(34): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(34): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(35): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(35): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(36): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(36): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(37): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(37): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(38): error C4430: missing type specifier - int assumed. Note: C++ does not
support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
d:\sample\node_modules\node-avro-io\node_modules\int64-native\src\Int64.h(38): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(21): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(22): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(22): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(25): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(26): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(26): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(25): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(29): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(30): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(30): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(29): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(33): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(34): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(34): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(33): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(37): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(38): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(38): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(37): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(41): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(42): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(42): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(41): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(45): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(46): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(46): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(45): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(49): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(50): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(50): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(49): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(53): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(54): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(54): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(53): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(57): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(58): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(58): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(57): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(61): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(62): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(62): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(61): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(65): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(66): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(66): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(65): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(69): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(70): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(70): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(69): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(73): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(74): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(74): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(73): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(77): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(78): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(78): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(77): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(81): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(82): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(82): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(81): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(85): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(86): error C2664: 'v8::FunctionTemplate::New' : cannot convert parameter 1 from 'v8::Handle<T> (__cdecl *)(const int)' to 'v8::Isolate *' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Value
          ]
          There is no context in which this conversion is possible
..\src\Int64.cc(86): error C2227: left of '->GetFunction' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(85): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(88): error C2660: 'v8::PersistentBase<T>::New' : function does not take 1 arguments [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          with
          [
              T=v8::Function
          ]
..\src\Int64.cc(89): error C2039: 'NewSymbol' : is not a member of 'v8::String' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
..\src\Int64.cc(89): error C3861: 'NewSymbol': identifier not found [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(122): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(122): error C2143: syntax error : missing ',' before '&' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(123): error C2248: 'v8::HandleScope::HandleScope' : cannot access protected member declared in class 'v8::HandleScope' [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(816) : see declaration of 'v8::HandleScope::HandleScope'
          C:\Users\james.terry\.node-gyp\0.12.7\deps\v8\include\v8.h(800) : see declaration of 'v8::HandleScope'
..\src\Int64.cc(125): error C2065: 'args' : undeclared identifier [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(125): error C2228: left of '.Length' must have class/struct/union [D:\sample\node_modules\node-avro-io\node_modules\int64-native\buil
d\Int64.vcxproj]
          type is ''unknown-type''
..\src\Int64.cc(127): error C2065: 'args' : undeclared identifier [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(127): error C2228: left of '.Length' must have class/struct/union [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
          type is ''unknown-type''
..\src\Int64.cc(128): error C2065: 'args' : undeclared identifier [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(128): error C2227: left of '->IsNumber' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(129): error C2065: 'args' : undeclared identifier [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(129): error C2227: left of '->ToNumber' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(130): error C2065: 'args' : undeclared identifier [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(130): error C2227: left of '->IsString' must point to class/struct/union/generic type [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(131): error C2065: 'args' : undeclared identifier [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
..\src\Int64.cc(131): fatal error C1003: error count exceeds 100; stopping compilation [D:\sample\node_modules\node-avro-io\node_modules\int64-native\build\Int64.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `msbuild` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "build"
gyp ERR! cwd D:\sample\node_modules\node-avro-io\node_modules\int64-native
gyp ERR! node -v v0.12.7
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok

> snappy@2.1.3 install D:\sample\node_modules\node-avro-io\node_modules\snappy
> node-gyp rebuild


D:\sample\node_modules\node-avro-io\node_modules\snappy>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  snappy-sinksource.cc
  snappy-stubs-internal.cc
  snappy.cc
  snappy.vcxproj -> D:\sample\node_modules\node-avro-io\node_modules\snappy\build\Release\\snappy.lib
  binding.cc
C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\xlocale(323): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc [D:\sample\node_modules\node-avro-io\node_modules\snappy\build\binding.vcxproj]
..\src\binding.cc(57): warning C4267: 'argument' : conversion from 'size_t' to 'int', possible loss of data [D:\sample\node_modules\node-avro-io\node_modules\snappy\build\binding.vcxproj]
..\src\binding.cc(84): warning C4267: 'argument' : conversion from 'size_t' to 'uint32_t', possible loss of data [D:\sample\node_modules\node-avro-io\node_modules\snappy\build\binding.vcxproj]
     Creating library D:\sample\node_modules\node-avro-io\node_modules\snappy\build\Release\binding.lib and object D:\sample\node_modules\node-avro-io\node_modules\snappy\build\Release\binding.exp
  Generating code
  Finished generating code
  binding.vcxproj -> D:\sample\node_modules\node-avro-io\node_modules\snappy\build\Release\\binding.node
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "node-avro-io"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE

npm ERR! int64-native@0.3.2 install: `node-gyp configure && node-gyp build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the int64-native@0.3.2 install script 'node-gyp configure && node-gyp build'.
npm ERR! This is most likely a problem with the int64-native package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp configure && node-gyp build
npm ERR! You can get their info via:
npm ERR!     npm owner ls int64-native
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\sample\sample\npm-debug.log
```

--- a/lib/datafile.js
+++ b/lib/datafile.js
@@ -267,7 +267,7 @@ _.extend(Reader.prototype, {
     _snappyDecompress: function(rawData, callback) {
         var compressedData = rawData.slice(0, rawData.length - 4);
         var checksum = rawData.slice(rawData.length - 4, rawData.length);
-        snappy.decompress(compressedData, snappy.parsers.raw, function(err, data) {
+        snappy.uncompress(compressedData, function(err, data) {
             if (err) return callback(err);
             var calculatedChecksum = crc32(data);
             if (calculatedChecksum.readUInt32BE(0) !== checksum.readUInt32BE(0))

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "lodash": "^2.4.1",
     "snappy": "^2.1.1",
     "buffer-crc32": "^0.2.5",
-    "int64-native": "0.3.2"
+    "int64-native": "^0.4.0"
   },
   "devDependencies": {
     "mocha": "^2.1.0",

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "lodash": "^2.4.1",
-    "snappy": "^2.1.1",
+    "snappy": "^4.0.1",
     "buffer-crc32": "^0.2.5",
     "int64-native": "^0.4.0"
   },

